README.md

```diff
@@ -33,7 +33,9 @@ Run with:
 sot
 ```
 
-!["Preview"](https://raw.githubusercontent.com/anistark/sot/refs/heads/main/images/sot-demo.gif)
+# Preview
+
+See the [`images/`](images/) folder for screenshots of the tool in action.
```

Comment:
It's better to have img rendered on readme. It also shows up on pypi page. Easier to check it out. Anyway, can get it added back separately...